src/App/Pages/Send/SendAddEditPage.xaml

```diff
@@ -252,28 +252,33 @@
                         </StackLayout>
                         <StackLayout
                             Orientation="Horizontal"
-                            Spacing="0">
+                            Spacing="0"
+                            AutomationProperties.IsInAccessibleTree="True"
+                            AutomationProperties.Name="{Binding OptionsAccessilibityText}">
+                            <StackLayout.GestureRecognizers>
+                                <TapGestureRecognizer Tapped="ToggleOptions_Clicked"/>
```

Comment:
Given that you're modifying this, could you change this to use `Command` instead of the event now?

Reply:
Sure i'll fix that.